src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/Cells/QRCodeCellViewModel.swift

```diff
@@ -30,6 +30,7 @@ struct QRCodeCellViewModel {
 			with: vCardV4(revDate: revDate),
 			encoding: .utf8,
 			size: CGSize(width: 280.0, height: 280.0),
+			scale: 1,
```

Comment:
```suggestion
			scale: UIScreen.main.scale,
```

As this is rendered for the screen, I think it should be set to screen scale

Reply:
After testing both implementations, i could not determine any difference. But i adopt your suggest because it makes more sense :)